README.md

```diff
@@ -121,6 +121,251 @@ curl https://minikube/atlas-contacts-app/v1/contacts -d '{"first_name": "Bob", "
 curl https://minikube/atlas-contacts-app/v1/contacts?_filter='first_name=="Mike"'
 ```
 
+##### Pagination
+
+Contacts App implements pagination in by adding application specific page token implementation.
+
+Actually the service supports "composite" pagination in a specific way:
```

Comment:
If this is different than the default "recommended" way, you should put a disclaimer that it does it for demonstration purposes and shouldn't be emulated.

If this is something that _is_ a good idea, should we support it as the default behavior? What is our default behavior, anyway?

Reply:
Good call.